inc/commondbtm.class.php

```diff
@@ -226,6 +226,8 @@ function post_getFromDB() {
     * Actions done to not show some fields when geting a single item from API calls
     *
     * @param array $fields Fields to unset undiscloseds
```

Comment:
The method signature is visible in the doc, so no need to add the `&` in the docblock (current api doc is not up-to-date, I've requested the update, I'll give you the link to the result once done).